### ISSUE TITLE
css_framework='boostrap3' or css_framework=''bootsrap4' or css_framework='bootstrap' and bs_version=4 not working

### ISSUE THREAD

**Issue by Viraaj-A** [lixxu/flask-paginate#107]
No matter what combination of the following I try, the pagination links are still horizontal: 

```
css_framework='boostrap3' / 'boostrap3_3' / 'boostrap4'
bs_version = 3 / 4  
```
`pagination = Pagination(css_framework='bootstrap3', page=page, per_page=per_page, total=len(ecli_list))`

Do I have to do something else then simply pass the bootstrap parameter into the pagination function?

**Comment by lixxu** [comment]
do you have a typo error: `boostrap`?

**Comment by Viraaj-A** [comment]
No, sorry I made the typo error in the above but not in the code. You can see the other examples do not have a typo error. 

I was able to get around the issue by a combination of both setting the css_framework='bootstrap3' and then manually changing the relevant list elements in the CSS file. I
